src/main/drivers/timer_def.h

```diff
@@ -644,3 +644,127 @@
 #define DEF_TIM_AF__PI7__TCH_TIM8_CH3     D(3, 8)
 
 #endif
+
+
+#ifdef USE_TIMER_MGMT
+
+#define TIMER_TAG_INDEX(t)           (t >> 3)
```

Comment:
Agreed, or just use an inline function, it shouldn't have flash-space footprint with modern GCC.